repository/src/test/java/uk/gov/hmcts/ccd/definition/store/repository/AccessTypeRolesRepositoryTest.java

```diff
@@ -22,7 +22,7 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeLiteEntity.toCaseTypeLiteEntity;
 
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
I think the ExtendWith(SpringExtension.class) annotation can be removed as it's now included as a meta-annotation on SpringBootTest (see https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#junit-5).